controllers/consoleplugin/consoleplugin_test.go

```diff
@@ -91,17 +95,22 @@ func TestServiceUpdateCheck(t *testing.T) {
 
 	//equals specs
 	serviceSpec, containerConfig := getServiceSpecs()
-	assert.Equal(serviceNeedsUpdate(&serviceSpec, &containerConfig), false)
+	// assert.Equal(serviceNeedsUpdate(&serviceSpec, &containerConfig, testNamespace), false)
```

Comment:
shouldn't that pass?

Reply:
Yes, I commented it when debugging and forgot t uncomment it.

Reply:
Side question here, the reason I commented this check is because I wanted to see the result of the following part of the test.

Should we split this test into different one ?

Reply:
yes go ahead if it's better :-)